Website docs page sidebar reorganization and restyle (#1603)
vmware-tanzu/pinniped: site/content/docs/howto/idps/_index.md

---
title: How-to Guides for Configuring IDPs
cascade:
  layout: docs
menu:
  docs:
    name: Supervisor Configuration
    identifier: howto-configure-idps
    weight: 60
---

> These how-to guides show you how to install and configure Pinniped with specific identity providers:

Reviewer: ```suggestion
These how-to guides show how to configure the Pinniped Supervisor after it is installed:
```

Author: 👍
